frontend/src/components/alert/snackBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+import Fade from '@material-ui/core/Fade';
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const SnackbarAlerts = props => {
+  const { open, message, severity, handleClose } = props;
```

Comment:
You'll need to make a TS interface and make `props` of that type to avoid the PropTypes error that eslint is giving.

TS interface: https://www.typescriptlang.org/docs/handbook/2/objects.html#interfaces-vs-intersections
Link to deprecated doc of TS interface but might still be useful: https://www.typescriptlang.org/docs/handbook/interfaces.html